lib/errors.ts

```diff
@@ -149,6 +149,8 @@ const EXPECTED_ERROR_REGEXES = [
 	/^BalenaDeviceNotFound/, // balena-sdk
 	/^BalenaExpiredToken/, // balena-sdk
 	/^BalenaInvalidDeviceType/, // balena-sdk
+	/Cannot deactivate devices of Starter applications.$/, // balena-sdk
```

Comment:
I guess you meant for the final dot `.` to match the error message:
```
BalenaRequestError: Request error: Cannot deactivate devices of Starter applications.
```
But `.` has a special meaning in the regex, which matches `.` and any other character, so you might have wanted to escape it as `\.`. But really what I would like to see is more generalisation that avoided us having to change the CLI for small changes to the error message. Other than the error codes discussed in [flowdock](https://www.flowdock.com/app/rulemotion/i-cli/threads/bcx7YUa-8fuhJ2lX55sd8sNuRCX), we could at least use case insensitive matching (`/i` flag) and avoid including the application type like `Starter`. I'd suggest:
```
/Cannot deactivate devices/i,
```
If that much matches, changes are we don't want to report it to Sentry.io.

Re comment `// balena-sdk`, perhaps it should be `// balena-api`, because the error message cannot be found in the `balena-sdk`.